### PR TITLE
Add support to include Java Flight Recorder (JFR) parameters and SHUTDOWN_WAIT_TIME_MS parameter

### PR DESCRIPTION
**Pull request by swatisawant**
## Changes proposed in this pull request

- Added support to include Java Flight Recorder (JFR) parameters for collecting diagnostic and profiling data while launching server and leader members.
- Added support in framework to include gemfire.CacheServerLauncher.SHUTDOWN_WAIT_TIME_MS parameter while launching lead and server member.

## Patch testing

Verified the changes by running northwind bt

## ReleaseNotes.txt changes

NA

## Other PRs 

No

### CODE REVIEW

File: dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java [hunk 1]
```diff
@@ -2285,7 +2287,8 @@ protected void killVM(String vmDir, String clientName, boolean isLead) {
         Log.getLogWriter().info(clientName + " stopped successfully...");
     }
 
-    protected void startVM(String vmDir, String clientName, boolean isLead) {
+    protected void startVM(String vmDir, String clientName, boolean isLead) throws InterruptedException {
+        Thread.sleep(100000);
```
Comment (suranjan) [inline dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java]: What is this change for?

Comment (suranjan) [inline dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java]: Rest looks ok.

Comment (swatisawant) [inline dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java]: We had a discussion about the SNAP-1177 and at that time, you had suggested to make sure of restarting the member after certain delay. That's the reason, I added this delay of 100 secs before calling start for the member which is getting recycled.

Comment (suranjan) [inline dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java]: If we use SHUTDOWN_WAIT_TIME then is it needed?

Comment (swatisawant) [inline dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java]: No. It will also increase my test time. Will remove it and merge the changes.

**Pull request merged**